Server/GameModel.cs

```diff
@@ -190,14 +191,16 @@ void StartSpawnTimer()
             timer.Start();
         }
 
+        RPC.Item GetRpcItem(Item item)
+        {
+            var rpcItem = new RPC.Item(item.Id, item.Type, item.Position);
+            return rpcItem;
+        }
         void Environment(string id)
         {
-            var itemsRpc = new List<RPC.Item>();
-            foreach (var item in items.Values)
-            {
-                var itemRpc = new RPC.Item(item.Id, item.Type, item.Position);
-                itemsRpc.Add(itemRpc);
-            }
+            var itemsRpc = items.Values
```

Comment:
```suggestion
            var rpcItems = items.Values
```
他の場所でも使ってるしこっちでよいかも？